test_accounts/appeals_apis/appeals_status_test_accounts.md

```diff
@@ -0,0 +1,45 @@
+# Sandbox Test Accounts for Appeals Status API (v1)
+
+This document provides test user accounts for the [Appeals Status API (v1)](https://developer.va.gov/explore/api/appeals-status/docs?version=current). The API will return synthetic VA records for all test accounts.
```

Comment:
This link currently takes the consumer to the v0 documentation, but that will not be the case once the v1 documentation is taken live.